packages/terra-tag/tests/nightwatch/MultipleTags.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import IconBookmark from '../../../terra-icon/lib/icon/IconTile';
+import Tag from '../../lib/Tag';
+
+const icon = <IconBookmark />;
+const text = 'At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores ';
+
+const IconTag = () => <Tag icon={icon} text="Icon Tag" id="iconTag" />;
+const LongTextIconTag = () => <Tag icon={icon} text={text} id="longTextIcon" onClick={() => 1} />;
+const DefaultTag = () => <Tag id="default" text="Default Tag" />;
+const OnClickTag = () => <Tag id="on-click" text="Clickable Tag" onClick={() => 1} />;
```

Comment:
Our solution designers will be testing the onClick functionality of these tags, so I say it's ok to have an alert display something so that they can verify the functionality.

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/b04c1661bc417ddbf31b7da23c28d95ec857e1df).